docs/ConfigurationFile.md

```diff
@@ -162,6 +162,8 @@ exports.config = {
     //
     // The number of times to retry the entire specfile when it fails as a whole
     specFileRetries: 1,
+    // Retried specfiles are inserted at the beginning of the queue.
+    specFileRequeue: 'first',
```

Comment:
I don't like string here, because it's obvious what should be here. You will go to the documentation each time. If we have only two states it could be boolean true or false. For example retriesRequeueAtBegginning: true or false by default.

Reply:
I agree - I cannot imagine any reason why retries would ever be inserted somewhere other than at the very beginning or very end. I did it this way mainly because I could not come up with a non-clumsy name for the boolean setting at the time.

The term "defer" is a nice boolean descriptor for now vs later - how about something like `specFileRetriesDeferred`? Then "true" means defer retries to the end of the queue and "false" means don't defer and retry now.

I also considered changing `specFileRequeue` into an object that could group all current and future retry-related settings, for example:
```js
specFileRetries: {
    count: 1,
    defer: true
}
```
We could add backwards compatibility by transforming `specFileRetries: 1` to `specFileRetries: { count: 1 }` internally.
What are your thoughts on this?

Reply:
1) I like new name specFileRetriesDeferred
2) I don't like transform it to object, because it would affect to much users.
So, in my opinion would be enough to rename property and set boolean inside :)